.github/workflows/ci.yml

```diff
@@ -0,0 +1,51 @@
+on: [push, pull_request]
+
+jobs:
+  lint:
+    name: Linter
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: psf/black@stable
+        with:
+          options: "--check --verbose"
```

Comment:
I also prefer to add `-l 120`. By default, it is using classic 80 symbols per line which are outdated with widescreen monitors.

Reply:
I looked into one of your repositories. Added. Thanks, i ll remember it